hardly/handlers/distgit.py

```diff
@@ -303,7 +313,7 @@ def run(self) -> TaskResults:
             # If there was a new commit pushed before the pipeline ended, the report
             # might be incorrect until the new (for the new commit) pipeline finishes.
             commit_sha=source_git_pr.head_commit,
-            pr_id=source_git_pr.id,
```

Comment:
@jpopelka are you sure that removing `pr_id` here is correct?

Reply:
It's OK, we use the reporter here just for commit comments so binding it to PR is not needed. I already tested it on stg and works OK.